src/components/EmailScreen.tsx

```diff
@@ -1,51 +1,91 @@
-import { BodyText, CaptionText } from 'components/Text'
+import { CaptionText } from 'components/Text'
+import AnonFace from 'icons/AnonFace'
 import Button from 'components/Button'
 import EmailScreenParams from 'models/EmailScreenParams'
 import classnames, {
   alignItems,
+  blur,
   display,
   flex,
   flexDirection,
   gap,
+  inset,
   justifyContent,
+  maxHeight,
+  minHeight,
+  minWidth,
+  objectFit,
+  overflow,
+  padding,
+  position,
+  zIndex,
 } from 'classnames/tailwind'
 import isMobileDevice from 'helpers/isMobileDevice'
 import openAppEmailLink from 'helpers/openAppEmailLink'
 
+const videoContainer = classnames(
+  position('absolute'),
+  inset('inset-0'),
+  blur('blur-xl'),
+  minWidth('min-w-full'),
+  minHeight('min-h-full'),
+  maxHeight('max-h-full'),
+  overflow('overflow-hidden'),
+  objectFit('object-cover'),
+  zIndex('-z-1')
+)
 const wrapper = classnames(
   display('flex'),
   flex('flex-1'),
   flexDirection('flex-col'),
   justifyContent('justify-center'),
   alignItems('items-center'),
-  gap('gap-y-4')
+  gap('gap-y-6'),
+  padding('px-4'),
+  zIndex('z-20')
 )
 const caption = classnames(
   display('flex'),
   flexDirection('flex-col'),
   justifyContent('justify-center'),
   alignItems('items-center'),
-  gap('gap-y-2')
+  gap('gap-y-8')
 )
 
+function OpenKetlBlock({ domain, token }: EmailScreenParams) {
```

Comment:
Let's put into separate file

Reply:
There is no point in separating this part. It's not so huge and doesn't contain any separate logic. We can leave them together